docs/fsharp-cheatsheet.md

```diff
@@ -555,136 +555,158 @@ Another way of implementing interfaces is to use *object expressions*.
 <a name="CodeOrganization_Modules"></a>
 ### Modules
 Modules are key building blocks for grouping related code; they can contain `types`, `let` bindings, or (nested) sub `module`s.
```

Comment:
maybe namespaces needs to go first? Maybe you need to mention that everything lives in hierarchical namespaces with dots, and you can `open` namespaces, modules, types to bring stuff into scope to allow relative references?

Reply:
That was intentional; I put namespaces second so I had something to work with.

Reply:
There's definitely a chicken and egg aspect
I guess I'm looking for something to convey that:
- .NET itself only has types with dotted prefixes
- `module`s and `type`s are just two flavors of types
But, namespace and module have comonalities in terms of how you can use them a) within a source file b) in an overall system to group/nest things
It's pretty good relative to were it is and I do know that I'd turn the cheatsheet into a book very fast if I followed by natural tendencies when writing!